bin/copy-modules.js

```diff
@@ -21,6 +21,11 @@ feature.setConfig(envConfig);
 
 
 function ignoreFile(file) {
+  // We need to ignore all the files except the index.js
```

Comment:
Can we add some more details about why workers are not included (they are bundled) and why the index files are included (they are API entry points)